deps/chakrashim/include/v8.h

```diff
@@ -2068,6 +2069,40 @@ class V8_EXPORT SharedArrayBuffer : public Object {
   SharedArrayBuffer();
 };
 
+
+/**
+* A JSON Parser and Stringifier.
+*/
+class V8_EXPORT JSON {
+public:
+    /**
+    * Tries to parse the string |json_string| and returns it as value if
+    * successful.
+    *
+    * \param json_string The string to parse.
+    * \return The corresponding value if successfully parsed.
+    */
```

Comment:
remove comments. If you see around in this file, we don't retain the comments from `deps\v8\include\v8.h` file.